docs/docs/workflows/config.md

```diff
@@ -84,6 +84,13 @@ If you haven't generated a configuration file yet, you can do so by adding the `
 
 You must be authenticated with the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the shell session.
 
+Your Azure account also needs to have at least the following permissions:
```

Comment:
```suggestion
You must be authenticated with the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the shell session, and you need at least the following permissions:
```
nit: I'd shorten and link this to the previos sentence, but you can keep it if you prefer.